Authors:  Cecil Curry <leycec@gmail.com>
Upstream: no
Synopsis: Allow "CMAKE_BUILD_TYPE" to be set to "None", as the "cmake" exlib
currently enforces.
--- old/cmake/modules/CGAL_SetupFlags.cmake	2010-07-13 15:58:21.000000000 +0000
+++ new/cmake/modules/CGAL_SetupFlags.cmake	2010-11-17 03:58:45.376000070 +0000
@@ -57,10 +57,6 @@
   endif()
 endif()
 
-if ( NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Release" AND NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Debug" )
-  message( FATAL_ERROR "${CMAKE_BUILD_TYPE} is not a valid build type: only Release or Debug is allowed" )
-endif()
-
 message( STATUS "Build type: ${CMAKE_BUILD_TYPE}" )
 
 string( TOUPPER "${CMAKE_BUILD_TYPE}" CGAL_BUILD_TYPE_UPPER )
